Added BuildMultiTargeting props

devlead opened the pull request:
This makes this NuGet package compatible with multi targeted framework builds.

Testesd with and required for libgit2/libgit2sharp#1606

bording commented:
@devlead There's an easier way to handle this. Just have the `buildMultiTargeting` import the files from the `build` folder. That way they are static and never have to be updated.

devlead commented:
True, good catch👍 I'll add an import and revert script change when I'm back at a computer.

devlead commented:
@bording thing I've addressed your feedback, PR 

Tried libgit2/libgit2sharp#1606 with https://ci.appveyor.com/project/devlead/libgit2sharp-nativebinaries/build/1.0.2/artifacts and it still compiles 😉
